mbed_lstools/lstools_linux_generic.py

```diff
@@ -16,290 +16,85 @@
 """
 
 import re
```

Comment:
This file is a thing of beauty, well done! 👍

Reply:
Thank you. 😊